t/cobrand/merton.t

```diff
@@ -159,4 +160,41 @@ sub test_visit_problem {
     $mech->log_out_ok;
 }
 
+subtest 'check open311 inclusion of service value into extra data' => sub {
+
+    $merton->api_key('aaa');
+    $merton->endpoint('http://endpoint.example.org');
+    $merton->jurisdiction('j');
+
+    my $sender = FixMyStreet::SendReport::Open311->new(
+        bodies => [ $merton ], body_config => { $merton->id => $merton },
+    );
+
+    my $contact = $mech->create_contact_ok(
+        body_id => $merton->id,
+        email => 'FT',
+        category => 'Fly-tipping',
+    );
+
+    my ($new_report) = $mech->create_problems_for_body(1, $merton->id, 'Title', {
+        postcode => 'SM4 5DX', areas => ",2500,", category => 'Fly-tipping',
+        cobrand => 'merton', user => $normaluser, state => 'open', service => 'desktop'
+    });
+
+    FixMyStreet::override_config {
+        ALLOWED_COBRANDS => 'merton',
+    }, sub {
+        $sender->send($new_report, {
+            easting => 1,
+            northing => 2,
+            url => 'http://example.org/',
+        });
+    };
+
+    $new_report->update;
```

Comment:
This persists things in the tests, but not in the actual code, so the service attribute still isn't being persisted. This line needs to go in `perllib/FixMyStreet/Cobrand/Merton.pm` and the tests need a `$new_report->discard_changes` line here to ensure we're retrieving the latest row from the DB, rather than whatever is in memory.